content/en/blog/2024/inpod-traffic-redirection-ambient/index.md

```diff
@@ -0,0 +1,228 @@
+Getting broad support for Kubernetes anywhere has become the No. 1 requirement for ambient mesh moving to Beta — people have come to expect Istio to work on any Kubernetes platform and with any Container Network Interface (CNI) layer. After all, Ambient wouldn’t be ambient without being all around you!
+
+At Solo, we've been integrating ambient mode into our Gloo Mesh product, and came up with an innovative solution to this problem. We had decided
+to [upstream](https://github.com/istio/istio/issues/48212) our changes in late 2023 to help Ambient reach beta faster
+so more users can operate ambient in Istio 1.21 or newer and enjoy the benefits of ambient sidecar-less mesh in their
+platforms regardless of their preferred CNIs.
+
+## How did we get here?
+
+### Service meshes and CNIs: it's complicated
+
+Istio is a service mesh, and service meshes by definition are not *CNI implementations* - service meshes build on top of the
+low-level, standards-compliant CNI implementation present in Kubernetes clusters to offer the high-level policy and security
+controls users need at scale.
+
+Basically, before you can do things like secure pod traffic with mTLS and apply lots of high-level authentication and authorization policy
+at the service mesh layer, you must have a functional Kubernetes cluster with a functional CNI implementation, to make sure the
+basic networking pathways are set up so that packets can get from one pod to another (and from one node to another) in
+your cluster.
+
+Though some service meshes may also ship and require their own in-house CNI implementation, the Istio project has chosen not
+to, for maximum compatibility with managed offerings, cross-vendor support, and composability with the broader CNCF ecosystem.
+
+### Traffic redirection in ambient alpha
+
+The [istio-cni](/docs/setup/additional-setup/cni/) component is an optional component in the sidecar data plane mode,
+commonly used to remove the [requirement for the `NET_ADMIN` and `NET_RAW` capabilities](/docs/ops/deployment/requirements/) for
+users deploying pods into the mesh. 
+
+istio-cni is a required component in the ambient
+data plane mode. Whenever pods are added to an ambient mesh, the istio-cni component configures traffic redirection for all
+incoming and outgoing traffic between the pods and the [ztunnel](/blog/2023/rust-based-ztunnel/) running on
+the pod's node, via the node-level network namespace. The key difference between sidecar and ambient modes is that 
+the redirection in ambient is _across network namespaces_, from the application namespace to the Istio system namespace
```

Comment:
As we are using the term `namespace` both in the context of Linux networking namespace and K8s namespaces, it could be a bit confusing for the users. Can you rephrase it?

Reply:
Is this what is meant by "host network based " or "host-based" as mentioned in the next section?

Reply:
Tried to clarify this, PTAL